src/xsar/sentinel1_meta.py

```diff
@@ -831,6 +831,14 @@ def coords2heading(self, atracks, xtracks, to_grid=False, approx=True):
         _, heading = haversine(lon1, lat1, lon2, lat2)
         return heading
 
+    @property
+    def bursts(self):
```

Comment:
if bursts are not rasterisable, I think that we should name them as `self._bursts`, not `self.bursts`

Reply:
solved by f024c7420